NobodyXu/tokio
# Speedup CI

- Disable incremental build since ci always build from scratch
 - Use crates.io sparse registries to speedup fetching
 - Pass `--no-deps` to `cargo-clippy`

 - Use `cargo-nextest` in job test to speedup testing
 - Use `cargo-nextest` in job test-unstable to speedup testing
 - Use `cargo-nextest` in job test-unstable-taskdump to speedup testing
 - Use `cargo-nextest` in job asan to speedup testing
 - Use `cargo-nextest` in job no-atomic-u64 to speedup testing

 - Run `cargo check --benches` for `benches/` in job test Since the benchmark is not run
 - Run `cargo-check` instead of `cargo-build` in job test-parking_lot since no test is run
 - Run `cargo-check` instead of `cargo-build` in job no-atomic-u64



## Motivation



## Solution
